extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -217,7 +217,7 @@ export class AmpLiveList extends AMP.BaseElement {
         actualCount);
 
     if (isExperimentOn(this.win, 'amp-live-list-sorting')) {
-      this.isReverseOrder_ = this.element.hasAttribute('reverse');
+      this.isReverseOrder_ = this.element.gettAttribute('sort') === 'ascending';
```

Comment:
typo (I think?): gettAttribute -> getAttribute

Reply:
@ericlindley-g good catch! LOL

Reply:
Wishing I had a "Hackerman" GIF with my face photoshopped in that I could put here...